anime_downloader/sites/animeonline360.py

```diff
@@ -0,0 +1,54 @@
+import json, requests
+
+from anime_downloader.sites.anime import Anime, AnimeEpisode, SearchResult
+from anime_downloader.sites import helpers
+
+class AnimeOnline(Anime, sitename = 'animeonline360'):
+
+    sitename = 'animeonline360'
+
+    @classmethod
+    def search(cls, query):
+        try:
+            # r = helpers.get('https://animeonline360.me/wp-json/dooplay/search/?nonce=12d75e884b', params = {'keyword': query}).json()
+            r = helpers.soupify(helpers.get('https://animeonline360.me/', params = {'s': query})).select('div.title')
+            results = [{"title": x.text, "url": x.a['href']} for x in r]
+            search_results = [
+                SearchResult(
+                    title = i['title'],
+                    url = i['url'],
+                    meta= {})
+
+                for i in results
+                ]
+
+            return search_results
+        except:
+            return ""
+
+    def _scrape_episodes(self):

```

Comment:
You can replace this entire function with
```
    def _scrape_episodes(self):
        data = helpers.soupify(helpers.get(self.url)).select('div.episodiotitle > a')
        return [i.get('href') for i in data[::-1]]
```
Should work just as fine, no need to make things too complicated

Reply:
ok